### PR TITLE
setting up latest posts block

### LINKED ISSUE #17: Block: Latest Posts
Grid style of Latest Posts block:

Blocks: Latest Posts

@melchoyce Will the list view ever be used?

### PR DESCRIPTION
**Pull request by yingles**
Fixes #17 

@melchoyce Any thoughts on a hover state? I just went with `text-decoration: underline;` on hover and focus for now since the `a` is all that's clickable.

Screen Shot 2019-04-23 at 11 12 06 PM

### CODE REVIEW

File: sass/abstracts/_colors.scss [hunk 1]
```diff
@@ -6,6 +6,7 @@ $blue: #003b63;
 $red: #bd2b2e;
 $grey: #303030;
 $gray: $grey;
+$grey--alt: #353530;
```
Comment (ryelle) [inline sass/abstracts/_colors.scss]: We shouldn't have an alt, sometimes the sketch file handles color wrong - @melchoyce Should this be `#303030;`, or is this actually another color?

File: sass/elements/_latest-posts.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+	a {
+		@include font-size( 2.4 );
+		line-height: $font__line-height--small;
+		color: $white;
+		text-decoration: none;
+
+		&:hover,
+		&:focus,
+		&:visited {
```
Comment (ryelle) [inline sass/elements/_latest-posts.scss]: We have the `hover-state` mixin that handles this, so you can do 

```scss
@include hover-state {
    // your styles
}
```

File: sass/elements/_latest-posts.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+		}
+
+		&:hover,
+		&:focus {
+			text-decoration: underline;
+		}
+	}
+
+	time {
+		color: rgba(255, 255, 255, 0.7);
```
Comment (ryelle) [inline sass/elements/_latest-posts.scss]: This is a nitpick, but can you do `rgba($white, 0.7);` instead? Just to stay consistent with the color vars.

**Comment by melchoyce** [comment]
> @melchoyce Any thoughts on a hover state? I just went with text-decoration: underline; on hover and focus for now since the a is all that's clickable.

Sounds good to me 👍

File: sass/abstracts/_colors.scss (hunk 1 above)
Comment (melchoyce) [reply sass/abstracts/_colors.scss]: Oops yeah, this should be #303030. Every color should match the existing variables in the theme.

Comment (yingles) [reply sass/abstracts/_colors.scss]: Meant to point that out in the PR and get your feedback. Glad you caught it. Went ahead and switched it.

File: sass/elements/_latest-posts.scss (hunk 2 above)
Comment (yingles) [reply sass/elements/_latest-posts.scss]: Updated to use the mixin.

File: sass/elements/_latest-posts.scss (hunk 3 above)
Comment (yingles) [reply sass/elements/_latest-posts.scss]: Not a problem. :)

**Pull request merged**